src/plugins/jqplot.donutRenderer.js

```diff
@@ -393,7 +393,10 @@
         else {
             this._thickness = this.thickness || mindim / 2 / (this._numberSeries + 1) * 0.85;
         }
-
+        if (this._diameter < 6) {
```

Comment:
This issue occurred only in donut renderer and pie renderer was working fine, as minimum diameter was already defined. Refer - d plugins/jqplot.pieRenderer.js line 462.
I thought both pie and donut chart are quite similar and decided to use same value "6" as minimum diameter for donut chart as well.